apps/paymaster-proxy/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "@eth-optimism/paymaster-proxy",
+  "version": "0.0.0",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 0",
```

Comment:
fails with 0 to prevent this app from blocking release to other apps in the monorepo